recipes/napari/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "napari" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed.
+  # url: https://github.com/napari/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Please remove extra comment lines too

Reply:
Thanks! I kinda wanted to leave the ones that I thought I might refer to in the future, eg if we move to a GitHub-centric release process. Is it ok if I leave it around for future maintainers?

Reply:
This was done.